src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -555,6 +555,28 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
     runTransaction(action)
   }
 
+  /**
+    * Updates the existing input symbols to replace expressions with real values.
+    * @return The number of rows updated - as a Future.
```

Comment:
Do you actually check that the number of rows returned is correct / incorrect ? Or is it just for logging purposes ?